phpunit/functional/LogTest.php

```diff
@@ -826,4 +826,73 @@ public function testGetHistoryDataRawTimestamp()
             $this->assertSame('2023-11-01 00:00:00', $entry['date_mod']);
         }
     }
+
+    /**
+     * Test that old_id and new_id values are correctly saved in logs
+     */
+    public function testOldIdNewIdValues()
+    {
+        global $DB;
+
+        $computer = new \Computer();
+        $computers_id = $computer->add([
+            'name'        => 'Test computer',
+            'entities_id' => getItemByTypeName('Entity', '_test_root_entity', true)
+        ]);
+        $this->assertGreaterThan(0, $computers_id);
```

Comment:
There is a `createItem()` function which checks the creation process in more detail.

```suggestion
        $computers_id = $this->createItem('Computer', [
            'name'        => 'Test computer',
            'entities_id' => getItemByTypeName('Entity', '_test_root_entity', true)
        ])->getID();
```

_Same thing with `updateItem()`_